### DISCUSSION TITLE
Custom TV Scanner: invalid provider

### DISCUSSION THREAD

**Discussion by fnwc** [Taxel/PlexTraktSync#577] [Support] [answered]
I'm unable to sync my TV show watched status due to using a custom TV agent:

`ERROR: <Show:746471:The-Wire>: Unable to parse a valid provider from guid com.plexapp.agents.xbmcnfotv://79126?lang=xn`

Strangely enough, I use a similar custom agent for movies, but they sync without issue. 

This looks like it might have been addressed in #144 but I'm not seeing the same behavior between movies/TV shows.

**Comment by simonc56** [comment, marked as answer]
Duplicate of :
- https://github.com/Taxel/PlexTraktSync/issues/452

Please continue in this appropriate issue.
Also give as much details as you can (version of script, os, config file, etc...)

**Reply by fnwc** [reply]
Hi twolaw, I'm actually using your docker here https://hub.docker.com/r/twolaw/plextraktsync/tags (latest).

This is running off of Unraid in the docker terminal. I haven't changed anything in the config from the docker defaults, and I'm using the same custom agent you are in the duplicate ticket: XBMCnfoTVImporter (V 1.1-72).

**Reply by simonc56** [reply]
Don't mix up topics please.
It you want to talk about my fork (which is not updated), do it on my repository.
As stated in readme file, the official PlexTraktSync docker version is `ghcr.io/taxel/plextraktsync`
